nixos/modules/services/networking/wireguard.nix

```diff
@@ -137,12 +137,12 @@ let
   generateConf = name: values: pkgs.writeText "wireguard-${name}.conf" ''
```

Comment:
Maybe, maybe not. Can you bring that up as a separate pull request in a separate issue? This one is about the upcoming version bump and preshared-key changes, not about some initiative to rewrite the whole module.